mlflow/server/js/craco.config.js

```diff
@@ -70,7 +70,7 @@ function configureIframeCSSPublicPaths(config, env) {
         .filter((oneOf) => oneOf.test?.toString() === /\.css$/.toString())
         .forEach((cssRule) => {
           cssRule.use
-            ?.filter((loaderConfig) => loaderConfig?.loader.match(/\/mini-css-extract-plugin\//))
+            ?.filter((loaderConfig) => loaderConfig?.loader.match(new RegExp(`\\${path.sep}mini-css-extract-plugin\\${path.sep}`)))
```

Comment:
```
            ?.filter((loaderConfig) => loaderConfig?.loader.split(path.sep).some(p => p === "mini-css-extract-plugin"))
```

Can we split by `path.sep` and find `"mini-css-extract-plugin"`?

Reply:
sure that works too, I'll slice the split to make sure `mini-css-extract-plugin` is an intermediate folder